packages/react/stories/index.js

```diff
@@ -41,6 +41,9 @@ storiesOf('props', module)
   .add('loading', () =>
     createMicrolink({ url: 'https://microlink.io', loading: true })
   )
+  .add('useIntersection', () =>
+    urls.map(url => createMicrolink({ url, useIntersection: true }))
```

Comment:
what do you think to use `lazy` as naming and use `true` by default?

Reply:
Yeah, the current name is horrible 😅. I'm not sure about the default `true` though, wouldn't it be better for users to opt-in to it?

Reply:
I've renamed the prop to `lazy` @Kikobeats but the default is still `false`